chart/values.yaml

```diff
@@ -644,13 +644,13 @@ config:
   core:
     dags_folder: '{{ include "airflow_dags" . }}'
     load_examples: 'False'
-    colored_console_log: 'False'
     executor: '{{ .Values.executor }}'
-    remote_logging: '{{- ternary "True" "False" .Values.elasticsearch.enabled }}'
   # Authentication backend used for the experimental API
   api:
     auth_backend: airflow.api.auth.backend.deny_all
   logging:
+    remote_logging: '{{- ternary "True" "False" .Values.elasticsearch.enabled }}'
```

Comment:
Where possible, we try to be compatible with Airflow 1.10 and 2.0 at the same time. I think we need to duplicate these keys. See: https://github.com/apache/airflow/blob/833c53b6a4ebd580c0e5daf75215d38e875d5ddd/chart/values.yaml#L668

Reply:
Done!